src/euphorie/client/browser/templates/webhelpers.pt

```diff
@@ -213,13 +213,24 @@
                      name="measure-custom-${measure/solution_id}"
                      type="checkbox"
               />
-              <textarea autofocus="autofocus"
-                        cols="50"
-                        name="measure-custom-${measure/solution_id}"
-                        placeholder="${view/placeholder_add_extra}"
-                        rows="3"
-                        tal:content="structure python:webhelpers.get_safe_html(measure['text'])"
-              ></textarea>
+              <div class="pat-rich-editor pat-rich toolbar-detached">
+                <textarea class="pat-tiptap"
+                          autofocus="autofocus"
+                          cols="50"
+                          name="measure-custom-${measure/solution_id}"
+                          placeholder="${view/placeholder_add_extra}"
+                          rows="3"
+                          data-pat-tiptap="
+                            toolbar-external: #assessment-toolbar;
+                            source-panel: #pat-modal.panel-html-source-editor;
+                            image-panel: #pat-modal.image-picker;
+                            link-panel: #pat-modal .link-panel;
+                            context-menu-link: #tiptap-context-menu-hyperlink-fieldname-comments
+                            url-scheme-mentions: https://quaive.cornelis.amsterdam/users/{USER};
```

Comment:
Is this normal?

Reply:
IIUC it will be dealt with in https://github.com/syslabcom/scrum/issues/707